Sources/Sentry/SentryCrashIntegration.m

```diff
@@ -273,10 +274,15 @@ + (void)enrichScope:(SentryScope *)scope crashWrapper:(SentryCrashWrapper *)cras
     [deviceData setValue:locale forKey:LOCALE_KEY];
 
 #if SENTRY_HAS_UIDEVICE && !defined(TESTCI)
-    // Acessessing UIScreen.mainScreen fails when using SentryTestObserver.
-    // It's a bug with the iOS 15 and 16 simulator, it runs fine with iOS 14.
```

Comment:
Does the fact that this is no longer the case mean that we can now remove the `!defined(TESTCI)` directive and run this logic in tests, if only for UI tests?

Reply:
True, I guess we could.